server/libs/request.js

```diff
@@ -17,12 +19,25 @@ const getHttpsClient = () => {
 
 function createClient({ baseURL, auth }) {
 
+    const logger = getLogger();
+
     const client = axios.create({
         baseURL,
         auth,
         httpsAgent: isProduction ? getHttpsClient() : null
     });
 
+    //Retry 3 times with 5 second gaps
+    axiosRetry(client, { retries: 3, retryDelay: () => { return 5000; } });
```

Comment:
Worth having these configurable? If we have an issue in production, we can easily then turn this off?